week-3/F-array-forEach/exercise.js

```diff
@@ -8,6 +8,17 @@
 */
 
 var arr = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15];
+arr.forEach(function(number) {
```

Comment:
Well done! This is correct 🎉 FizzBuzz is a famous exercise for testing your programming knowledge 🙂